hack/roll_nodes.sh

```diff
@@ -72,7 +72,7 @@ done
 for NODE in $NODES_TO_ROLL
 do
   detach $NODE
-  kubectl drain --delete-local-data --ignore-daemonsets $NODE
+  kubectl drain --delete-emptydir-data --ignore-daemonsets $NODE
```

Comment:
I can't find this as a flag. The only option I've seen is --delete-local-data=false ?
https://github.com/kubernetes/kubernetes/issues/80228
Can you explain what this is doing (in the PR).

Reply:
There is one PR linked in that issue

https://github.com/kubernetes/kubernetes/pull/95076

Basically, only the name of this flag is changed. What it does is unchanged. The original flag (`--delete-local-data`) seems to be misleading, that's why the new flag `--delete-emptydir-data` was introduced.

Reply:
That PR is targeting the 0.20.0 release of kubernetes, should we introduce this change with our 0.20.0 release?

Reply:
> That PR is targeting the 0.20.0 release of kubernetes, should we introduce this change with our 0.20.0 release?

Right, maybe it's better to align the version with our release version.

So shall we revisit this PR when we're upgrading to v0.20.0 then?

Reply:
Yes sounds good to me, I think it's cleaner to align client with server versions